docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -67,14 +67,15 @@ Each action uses a connector, which provides connection information for a {kib}
 After you select a connector, set the action frequency. If the rule type supports alert summaries, you can choose to create a summary of alerts on each check interval or on a custom interval. For example, if you create a metrics threshold rule, you can send email notifications that summarize the new, ongoing, and recovered alerts each day:
 
 [role="screenshot"]
-image::images/rule-flyout-action-summary.png[UI for defining rule conditions on an index threshold rule,500]
+image::images/rule-flyout-action-summary.png[UI for defining rule conditions on a metric threshold rule,500]
```

Comment:
```suggestion
image::images/rule-flyout-action-summary.png[UI for defining rule conditions on a metrics threshold rule,500]
```

Reply:
It appears the rule being created for this screen shot is still an index threshold rule.  In order to get the right action frequency UX shown for the screenshot, we need to change to use a metric threshold instead.  So, seems like we should rename `x-pack/test/screenshot_creation/apps/response_ops_docs/stack_alerting/index_threshold_rule.ts` to `.../metric_threshold_rule...`, and change the rule params to allow metrics rule to validate.

I assume that's the intention anyway.  I'll see if I can hack something together for the metric threshold rule params ... it's often not 100% straight-forward.

Reply:
Yeah, the whole flow is for index threshold except that one action-summary related screenshot. If you think we can switch to using a metrics threshold (or any other rule that supports action summaries) for the whole flow, that'd be great.

Reply:
Ah ... I missed the part about "the whole flow".  Turns out, the screen caps are now a mix between index threshold and metric threshold, but just metric threshold showing the action-y bits.  We can probably do the whole flow instead, but showing active alerts (like at the bottom of the page) is actually quite difficult to set up.

I think this is ok for now - nice to show the new summary options anyway!  Open up a new issue for us to track using a single rule for the whole flow, if you think that's important.  It's possible we may find an "easier" rule type to use for this as well ...